### ISSUE TITLE
TameList_Monstrum

### ISSUE THREAD

**Issue by DragonLady-89** [meldurson/AllTameable#5]
This tamelist needs an update. the monstrum deepnorth & ashlands mods do not exsist anymore

**Comment by meldurson** [comment]
True, they don't. After loading and spitting out a bunch of warnings it does not affect anything. Hopefully the creatures will be added back. Until then you can remove the lines of the creatures that cause the warnings.

**Comment by DragonLady-89** [comment]
I did something almost as good, I took your monstrum tamelist and split it up in two so that I have a tamelist for both, with the new creatures added, in the new deepnorth. I can send it to you if you like?
